### PR TITLE
labeller: add delete button

### LINKED ISSUE #2: delete annotation
In the labeller interface, we need a button for deleting the saved annotations of the current video.

### PR DESCRIPTION
**Pull request by stumash**
Changes:
```
labeller/
    backend/
        app.js   add POST /deleteAnnotation
        utils.js add s3utils.deleteObject(bucket, path)
    frontend/
        deploy.sh  aws profile=default
        api.js     add deleteVideoAnnotation(annotationPath)
        index.html add delete button
        index.js   add delete button logic, clean up videoList update
```

closes #2

### CODE REVIEW

File: labeller/backend/utils.js [hunk 1]
```diff
@@ -34,11 +34,9 @@ const getAllBucketKeys = async (bucket, prefix) => {
   do {
     const params = {
       Bucket: bucket,
-      Prefix: prefix
+      Prefix: prefix,
+      ...(NextContinuationToken? {ContinuationToken: NextContinuationToken}: {})
```
Comment (Davidster) [inline labeller/backend/utils.js]:
```suggestion
      ContinuationToken: NextContinuationToken
```

File: labeller/frontend/deploy.sh [hunk 2]
```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env bash
 
-aws_profile_flag="${1}" # e.g. --profile=john
+if [ -n "${1}" ]; then
+    aws_profile_flag="${1}" # e.g. --profile=john
+else
+    aws_profile_flag="--profile=default"
```
Comment (Davidster) [inline labeller/frontend/deploy.sh:6]: :heart_eyes:

File: labeller/frontend/index.js [hunk 3]
```diff
@@ -117,6 +112,36 @@ const handleSaveButtonClicked = async () => {
...
+    return alert("No video has been selected");
+  }
+
+  const selectedVideoIndex = $("#videoList").prop("selectedIndex");
+  const {videoUrl, annotationInfo} = vidUrls_annotInfos[selectedVideoIndex];
+  if (!annotationInfo) {
+    return alert("No saved markers to delete")
+  }
+
+  if (!prompt("Are you sure? (Type anything and press enter for yes)")) {
```
Comment (Davidster) [inline labeller/frontend/index.js]:
```suggestion
  if(!confirm("Are you sure?")) {
```

**Review by Davidster — COMMENTED** [review]
sexy

File: labeller/frontend/index.js (hunk 3 above)
Comment (stumash) [reply labeller/frontend/index.js]: disagree. what if they type 'no' or 'NO'. maybe I should just make it `type 'yes' to confirm`

File: labeller/backend/utils.js (hunk 1 above)
Comment (stumash) [reply labeller/backend/utils.js]: The way I have it, the property `ContinuationToken` is not added to the object if `NextContinuationToken` is `undefined`.

Your way, the object **will have the property**, but its value will be `undefined`

**Pull request merged**